eclipse-scout-core/src/tile/Tile.ts

```diff
@@ -36,6 +36,8 @@ export class Tile extends Widget implements TileModel {
     this.selected = false;
     this.selectable = false;
     this.plainText = null;
+    // Null to let TileGrid decide whether to enable animation
```

Comment:
Remark about the commit message of 6732927d0eb3f563e4a9e6080f1412770d37b4a1:
* There is an unnecessary space in front of the last line.